var/spack/repos/builtin/packages/libffi/package.py

```diff
@@ -18,6 +18,8 @@ class Libffi(AutotoolsPackage, SourcewarePackage):
     version('3.2.1', sha256='d06ebb8e1d9a22d19e38d63fdb83954253f39bedc5d46232a05645685722ca37')
 
     patch('clang-powerpc-3.2.1.patch', when='@3.2.1%clang platform=linux')
+    # ref.: https://github.com/libffi/libffi/pull/561
+    patch('powerpc-3.3.patch', when='@3.3')
```

Comment:
I'm usually a fan of using URL patches to not litter the Spack repository with all the patches we need. How about using https://github.com/libffi/libffi/commit/4f9e20ac51ce13d46fed3c869e1deb6d9bb89444.patch instead? Either way is fine for me, though.

Reply:
Uff, in that case this would have been a good idea since the patch is GitHub based. I am just avoiding this when it goes to less-uptime guaranteed sources.
Good idea, will take care next time.